packages/cms-editor/src/CmsEditor.tsx

```diff
@@ -25,6 +25,12 @@ function CmsEditor(props: EditorProps) {
   useEffect(() => {
     setContext(props.context);
```

Comment:
by the way this should no longer be done, if not needed.
In this case I think it should not be needed to save the context in a state. just use it from the props :)